tests/torch/fx/test_fast_bias_correction.py

```diff
@@ -0,0 +1,97 @@
+
+from nncf.common.factory import NNCFGraphFactory
+from nncf.experimental.torch.fx.transformations import apply_quantization_transformations
+from nncf.quantization.algorithms.fast_bias_correction.torch_fx_backend import FXFastBiasCorrectionAlgoBackend
+from nncf.torch.dynamic_graph.patch_pytorch import disable_patching
+from nncf.torch.model_graph_manager import OPERATORS_WITH_BIAS_METATYPES
+from tests.post_training.test_templates.test_fast_bias_correction import TemplateTestFBCAlgorithm
+
+
+class TestTorchFXFBCAlgorithm(TemplateTestFBCAlgorithm):
+    @staticmethod
+    def list_to_backend_type(data: List) -> torch.Tensor:
+        return torch.Tensor(data)
+
+    @staticmethod
+    def get_backend() -> FXFastBiasCorrectionAlgoBackend:
+        return FXFastBiasCorrectionAlgoBackend
+
+    def _get_fx_model(model: torch.nn.Module):
+        device = next(model.named_parameters())[1].device
+        input_shape = model.INPUT_SIZE
+        if input_shape is None:
+            input_shape = [1, 3, 32, 32]
+        ex_input = torch.ones(input_shape).to(device)
+        model.eval()
+        with disable_patching():
+            exported_model = capture_pre_autograd_graph(model, args=(ex_input,))
+        apply_quantization_transformations(exported_model)
+        return exported_model
+
+    @staticmethod
+    def backend_specific_model(model: torch.nn.Module, tmp_dir: str):
+        fx_model = TestTorchFXFBCAlgorithm._get_fx_model(model)
+        return fx_model
+
+    @staticmethod
+    def fn_to_type(tensor):
+        return torch.Tensor(tensor)
+
+    @staticmethod
+    def get_transform_fn():
+        def transform_fn(data_item):
+            tensor, _ = data_item
+            return tensor
+
+        return transform_fn
+
+    @staticmethod
+    def check_bias(model: torch.fx.GraphModule, ref_bias: list):
+        ref_bias = torch.Tensor(ref_bias)
+        nncf_graph = NNCFGraphFactory.create(model)
+        for node in nncf_graph.get_all_nodes():
+            if node.metatype not in OPERATORS_WITH_BIAS_METATYPES:
+                continue
+            bias_value = FXFastBiasCorrectionAlgoBackend.get_bias_value(node, nncf_graph, model)
+            bias_value = torch.flatten(bias_value.data).cpu()
+            # TODO(AlexanderDokuchaev): return atol=0.0001 after fix 109189
+            assert torch.all(torch.isclose(bias_value, ref_bias, atol=0.02)), f"{bias_value} != {ref_bias}"
+            return
+        raise ValueError("Not found node with bias")
+
+
+@pytest.mark.cuda
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="Skipping for CPU-only setups")
+class TestTorchCudaFBCAlgorithm(TestTorchFXFBCAlgorithm):
+    @staticmethod
+    def list_to_backend_type(data: List) -> torch.Tensor:
+        return torch.Tensor(data).cuda()
+
+    @staticmethod
+    def backend_specific_model(model: bool, tmp_dir: str):
+        fx_cuda_model = super(TestTorchCudaFBCAlgorithm, TestTorchCudaFBCAlgorithm)._get_fx_model(model.cuda())
```

Comment:
```suggestion
        fx_cuda_model = super()._get_fx_model(model.cuda())
```

Reply:
using super() causes problems due to the staticmethod i believe

Reply:
You are right, please do `TestTorchFBCAlgorith.backend_specific_model(model.cuda(), tmp_dir)`

Reply:
Alright

Reply:
Will that not be bad for the class hirearchy in case we ever add more classes in between?

Reply:
Ok, let's keep it as it is, but please remove arguments from `super` call

Reply:
But using `super` causes problems. Would you suggest I go ahead with `TestTorchFBCAlgorith.backend_specific_model(model.cuda(), tmp_dir)` anyways?

Reply:
Sorry for unclear comments above, think best choice for now is to move this function outside of the function (but left in the same file, `test_bias_correction_aglorihtm` and call it in both `backend_specific_model` methods with different params.

Reply:
Done